impeller/entity/contents/content_context.cc

```diff
@@ -28,7 +28,9 @@ void ContentContextOptions::ApplyToPipelineDescriptor(
   desc.SetSampleCount(sample_count);
 
   ColorAttachmentDescriptor color0 = *desc.GetColorAttachmentDescriptor(0u);
-  color0.format = color_attachment_pixel_format;
+  FML_CHECK(color_attachment_pixel_format.has_value())
```

Comment:
We dont' want to terminate the process on this error. Perhaps, like the case above with the blend modes, add a validation log with a default? It's fine if the default is invalid, pipeline generation would fail anyway.